index.js

```diff
@@ -175,9 +171,9 @@ module.exports = class Filter extends Plugin {
         this.dependencies = this.processor.initialDependencies(srcDir);
```

Comment:
`initialDependencies` is still using `srcDir`, isn't this violating our `input/output` encapsulation goals?

Reply:
srcDir is not used for `fs` operations as far as I understood. 
@chriseppstein can you confirm if the srcDir used to access the files in inputTree.

Reply:
https://github.com/broccolijs/broccoli-persistent-filter/blob/master/lib/dependencies.js#L188 is doing disk operations

Reply:
My bad. Missed that. I made that change.